scripts/lp-create-bug

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+"""Mark the provided launchpad bugs as fix-released.
+
+Update the bug with a comment.
+"""
+
+import argparse
+import sys
+import re
+from launchpadlib.launchpad import Launchpad
+
+BUG_IMPORTANCE = ['Low', 'Wishlist', 'Medium', 'High', 'Critical']
+BUG_STATUS = ['New', 'Confirmed', 'Triaged', 'In Progress', 'Fix Committed']
+
+def main():
+    parser = argparse.ArgumentParser()
+
+    subject_tmpl = "Fixed in {project} version {version}."
+    comment_tmpl = (
+        "This bug is believed to be fixed in {project} in version "
+        "{version}. If this is still a problem for you, please make a "
+        "comment and set the state back to New\n\nThank you.")
```

Comment:
These two templates can be dropped, right?

Reply:
done.